### DISCUSSION TITLE
custom component - Cannot read property 'getTiles' of undefined

### DISCUSSION THREAD

**Discussion by AlexLin-hub** [vuetifyjs/vuetify#14077] [Q&A] [answered]
when i use [v-autocomplete] to make custom component, appear this error message. (**Cannot read property 'getTiles' of undefined.**)
This's my component code 
```
// customAutocomplete.vue
<template>
  <v-autocomplete
    v-model="newValue"
    :items="items"
    item-text="text"
    item-value="value"
    :filter="fillter"
    :filled="filled"
    return-object
    @input="$emit('input', $event)"
  />
</template>

<script>
import { VAutocomplete } from "vuetify/lib";
export default {
  extends: VAutocomplete,
  data() {
    return {
      newValue: this.value,
    };
  },
  methods: {
    fillter(item, queryText) {
      return (
        item["text"]
          .toLocaleLowerCase()
          .indexOf(queryText.toLocaleLowerCase()) > -1 ||
        item["value"]
          .toLocaleLowerCase()
          .indexOf(queryText.toLocaleLowerCase()) > -1
      );
    },
  },
}
</script>
```

This is error message, when i change select.
![截圖 2021-08-23 上午10 32 19](https://user-images.githubusercontent.com/62922047/130382939-584e651d-9e4f-4cc6-a2e9-2c7cc927cdfc.png)

![截圖 2021-08-23 上午10 32 38](https://user-images.githubusercontent.com/62922047/130382965-a52d1de1-f304-480e-a768-4bd4daf6377d.png)

**Comment by Samuel29** [comment]
Hi!
I'm having exactly the same problem. did you figure out a workaround @AlexLin-hub ?
my versions: 
```sh
egrep -i '"(vuetify|vue)"' package.json
    "vue": "^2.6.11",
    "vuetify": "^2.4.0",
```

**Comment by Samuel29** [comment]
oops my package.json wasn't up-to-date.
current versions:

```sh
├── vue@2.6.14
├── vuetify@2.5.13
```

**Reply by AlexLin-hub** [reply]
the problem still there, i don't know how i can fix.

**Reply by hadise-shaker** [reply]
hi. i have same problem. any result?

**Reply by KaelWD** [reply, marked as answer]
https://stackoverflow.com/a/70064606/2074736

**Reply by pate313373** [reply]
What exactly do you want to say with the stackoverflow link?

**Reply by KaelWD** [reply]
> Do not combine extends with `<template>` though, that will override the render function and almost always break the component.
